src/buffer/assert.rs

```diff
@@ -46,7 +49,7 @@ macro_rules! assert_buffer_eq {
                 }
                 // shouldn't get here, but this guards against future behavior
                 // that changes equality but not area or content
-                assert_eq!(actual, expected, "buffers not equal");
+                assert_eq!(actual, expected, $message);
```

Comment:
This is just the fall through case (in case we didn't already fail for not equal buffers). The main failures will occur on lines 10 and 35.